### ISSUE TITLE
Review image alt text

### ISSUE THREAD

**Issue by gregvanl** [Microsoft/vscode-docs#155]
Markdown image ![Dockerfile IntelliSense](images/docker/dockerfileintellisense.png) braket text becomes the image alt= text on the generated website pages.
Review all image text for good non-duplicate image descriptions.

**Comment by gregvanl** [comment]
dockerfile.md uses the same image name for each image [Dockerfile IntelliSense].  These should be made unique and descriptive.

**Comment by gregvanl** [comment]
nodejs.md all image text is "Your first Node Express App"

**Comment by gregvanl** [comment]
vscecli.md is using numbers 1, 2, 3 for images.

**Comment by gregvanl** [comment]
Duplicates in aspnet5.md (DNX commands in palette)
